README.md

```diff
@@ -1,79 +1,88 @@
-- Or you can install what3words AutoSuggest plugin manually. Download the plugin Zip archive and uncompress it. Copy or upload the `3-word-address-validation-field` folder to the `wp-content/plugins` folder on your web server.
-- Activate the plugin. From the Dashboard, navigate to Plugins and click on the *"Activate"* link under the entry for What3words Searchbox.
-- Customise and configure the plugin; from the Dashboard, navigate to the *Settings / what3words AutoSuggest plugin* page or click on the *"Settings"* link from the Plugins page on the Dashboard.
-- Click on the *"Save what3words AutoSuggest plugin Settings"* button to preserve your chosen settings and options.
 
-## Configuration ##
+## Contents
 
-### Enabling the plugin in a post or page ###
+* [Installation](docs/install.md)
+* [Configuration](docs/configuration.md)
+* [FAQ](docs/faq.md)
 
-To enable the plugin on a post or page, simply add a text input field contained inside a form, and add the field’s id to the plugin’s Input Selector(s) configuration option as described below.
+### More about what3words
 
-### what3words API settings ###
+what3words is available in 45+ languages, with pre-assigned, standardised 3 word addresses that are much easier to remember than lat/lng coordinates.
 
-You’ll need a  what3words API key to use this plugin, as it authenticates and interacts with the [what3words API](https://developer.what3words.com/public-api).
-If you don’t have a what3words API key yet, you can quickly and easily sign-up and register for one. It will just take a few minutes to do!
+Find our full [developer documentation](https://developer.what3words.com).
 
-### General Settings ###
+Learn more about our [Privacy Policy](https://what3words.com/privacy) or [Terms & Conditions](https://what3words.com/terms).
 
-- Input Selectors (WordPress only) – the input fields that the plugin is configured to search for 3 word addresses, specified as a list of css selectors. Multiple fields can be specified using commas, e.g. “.class, #id”.
-- Input Placeholder Text – an optional placeholder to be displayed in the input fields that are configured in the Input Selector(s) option. We recommend providing an example what3words address here with “e.g.” to illustrate how a what3words address should be entered, for example, “e.g. index.home.raft”
 
-### WooCommerce Support ###
+### Credit
 
-The plugin also allows you to automatically add a 3 word address validation field to a WooCommerce checkout form and save the 3 word address against the customer’s order details. By enabling WooCommerce checkout integration, the `#shipping_w3w` and `#billing_w3w` selectors will be automatically added to the list of Input Selector(s).
+This project was extended from the [WordPress Plugin Boilerplate](docs/boilerplate.md) and it's source can be found [here](https://github.com/DevinVinson/WordPress-Plugin-Boilerplate).
```

Comment:
Minor but it should be `its source can be found` without the apostrophe

Reply:
Will update, thanks